examples/databricks/search_experiments.py

```diff
@@ -0,0 +1,88 @@
+"""
+Search for experiments on Databricks.
+
+How to run:
+$ python examples/databricks/search_experiments.py --host *****host***** --token *****token***** --user *****user*****
+
+See also:
+https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token
+"""
```

Comment:
I don't think the example should be specific to Databricks, since the `SearchExperiments` REST API is being implemented by the open source `mlflow server`. Can we make this more general - `examples/search_experiments.py`?

Reply:
Makes sense. I'll generalize the code and move the file in `examples`.